src/main/java/bbm/brickbreaker/Ball.java

```diff
@@ -7,27 +7,61 @@ public class Ball extends Ellipse2D.Double {
     private int angle;
     private final int width;
     private final int height;
+    private double x;
+    private double y;
 
-    public Ball(int angle, int width, int height) {
+    public Ball(int angle, int width, int height, double x, double y) {
         this.angle = angle;
         this.width = width;
         this.height = height;
+        this.x = x;
+        this.y = y;
+    }
+
+    public void setPosition(double newX, double newY) {
+        this.x = newX;
+        this.y = newY;
+    }
+
+    public double getX() {
+        return x;
+    }
+
+    public double getY() {
+        return y;
     }
 
     public double locationX() {
-        return getX() + Math.cos(angle) * VELOCITY;
+        return getX() + Math.cos(Math.toRadians(angle)) * VELOCITY;
     }
 
     public double locationY() {
-        return getY() + Math.sin(angle) * VELOCITY;
+        return getY() + Math.sin(Math.toRadians(angle)) * VELOCITY;
     }
```

Comment:
What do these methods do?

Reply:
It converts the angle to radians so it can properly calculate the sine and cosine of the angles

Reply:
They get the x and y locations of the ball based on the current x and y, the direction the ball is facing, and the velocity at which the ball is moving.

Reply:
Deserves a better method name then